Makefile

```diff
@@ -68,6 +68,8 @@ show-docker-images: ## Show all docker images and versions used in the helm char
 		--set global.baseDomain=foo.com \
 		--set global.authSidecar.enabled=True \
 		--set global.blackboxExporterEnabled=True \
+		--set global.customLogging.awsSecretName=dummy \
```

Comment:
You should probably also add these to the other target `show-docker-images-with-private-registry`. I'm sure there's a way we could DRY that up, but I'm not good enough with Makefile to know how without doing research.

Reply:
@rob-1126 one way to do this would be to have a values file that enables all of these, and reference it via a `-f enable_all_features.yaml` or something. This would be generally useful so I think it would be good to break that out. Maybe put that into `tests/enable_all_features.yaml`. WDYT? CC @mishah334 @andriisoldatenko

Reply:
also FYI @rob-1126 i'm not asking you to add that yaml feature if you don't want to, but you should at least put this into the custom registry make target.

Reply:
> @rob-1126 one way to do this would be to have a values file that enables all of these, and reference it via a `-f enable_all_features.yaml` or something. This would be generally useful so I think it would be good to break that out. Maybe put that into `tests/enable_all_features.yaml`. WDYT? CC @mishah334 @andriisoldatenko

If there will be more options to set then, Helm [values files](https://helm.sh/docs/chart_template_guide/values_files/), can help to organize it. We can also use it with pytest def during the redenering of helm chart them.